spec/02-integration/01-cmd/02-start_stop_spec.lua

```diff
@@ -1,17 +1,19 @@
 local helpers = require "spec.helpers"
 
+local KILL_ALL = "pkill nginx; pkill serf; pkill dnsmasq"
```

Comment:
Same value used fifty times. Hence a constant because it was getting painful to change them all.